docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -84,12 +82,12 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `DISABLE_ROUTER_LOG`: **false**: Mutes printing of the router log.
 - `CERT_FILE`: **custom/https/cert.pem**: Cert file path used for HTTPS.
 - `KEY_FILE`: **custom/https/key.pem**: Key file path used for HTTPS.
-- `STATIC_ROOT_PATH`: **./**Upper level of template and static files path.
+- `STATIC_ROOT_PATH`: **./**: Upper level of template and static files path.
```

Comment:
it would be useful to specify what `.` is relative to, in this case. Directory containing executable file ? Directory from which the command was run ?